src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx

```diff
@@ -94,15 +94,20 @@ We recommend sending the data to a non-production New Relic account where you ca
 
 Alternatively, you can use filtering on queries to distinguish between metrics that come from Metric Streams and those that come through polling. All metrics coming from Metric Streams are tagged with `collector.name='cloudwatch-metric-streams'`.
 
-### Migration steps [#migration-steps]
+### Migration to metric streams [#migration-steps]
 
-On a typical deployment, migrating from API polling to metric stream involves the following steps (we recommend trying this on a dev / staging environment first):
+We recommend that you first migrate from API polling to metric streams in a dev or staging environment. For typical deployments, follow these steps to migrate to AWS Cloudwatch metric streams:  
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > AWS** and click on **Add an AWS account** to link your AWS account with New Relic. You need to do this even if your AWS account is already linked with polling integrations.
-2. Make sure you complete the last step in the onboarding, which involves enabling AWS CloudWatch metric stream and the AWS Kinesis Data Firehose to push metrics to New Relic.
-3. Repeat step 2 for any additional AWS region you want to monitor, since AWS CloudWatch requires one stream per region.
-4. Ensure metrics are received from all connected regions and namespaces. This may take several minutes.
-5. Disable all unnecessary polling integrations in the previous AWS provider account. [Remember that some integrations still need to be enabled](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream#integrations-not-replaced-streams) since they aren't fully replaced by metric streams.
+1. Replicate the namespaces from polling to metric streams by going to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > AWS**. Click on the **Migrate to AWS Cloudwatch metric streams** button, then configure your AWS Metric Streams account. 
+2. Download the customized CloudFormation template. This template contains the preconfigured namespaces that are based on your existing polling configuration.
+3. Add your account details. This lets you get the migration complete easily.
+
+**Alternate option:** 
```

Comment:
Hi @vinrelic -- when is this option a good option for users? What circumstance do we recommend the alternative migration?

Reply:
Alternative option is for customers who prefer not to use the CloudFormation template.